src/components/esphome-search.ts

```diff
@@ -29,7 +29,8 @@ export class ESPHomeSearch extends LitElement {
     css`
       esphome-text-input {
         --mdc-theme-primary: var(--primary-text-color);
-        width: 100vw;
+        width: calc(100% + calc(100vw - 100%));
```

Comment:
The problem is the scroll bar, but shouldn't just 100% width solve that?

Reply:
Unfortunately no. The scrollbar persists. I think it's because of inherited margins or padding from a parent container. I'll ask the UX folks tomorrow at work if there's a cleaner way to solve it.

Reply:
I stand corrected. I swear I tried that last night but it's working fine now. Adjusted PR accordingly.